webapp/src/app.tsx

```diff
@@ -1976,12 +1976,23 @@ export class ProjectView
                 cfg.dependencies[ext.name] = `workspace:${ext.id}`;
             }
         }
-        if (options.tsOnly) {
```

Comment:
IMO, add code that converts tsOnly to the ``typescript-only`` on the fly, to handle legacy scripts.